kokoro/macos-clang-release/build.sh

```diff
@@ -20,6 +20,9 @@ set -e
 # Display commands being run.
 set -x
 
+# Fix dubious ownership issues on Kokoro.
+git config --global --add safe.directory /tmpfs/src/github/SPIRV-Tools
+
```

Comment:
This just calls `kokoro/scripts/macos/build.sh`? Do you need the change here and there?  The same for `kokoro/macos-clang-debug/build.sh`.

Reply:
yes, missed this one, correct.